spec/lib/msf/ui/console/command_dispatcher/core_spec.rb

```diff
@@ -18,6 +18,8 @@
   def set_and_test_variable(name, framework_value, module_value, framework_re, module_re)
     # set the current module
     allow(core).to receive(:active_module).and_return(mod)
+    # always assume the variable is valid
+    allow(core).to receive(:tab_complete_option_names).and_return([ name ])
```

Comment:
This comment doesn't describe which variable we are referring to here or why we should consider it valid. Also I thought the point here was to provide additional validation so curious why we would change the code to assume the variable is valid here?